src/mscorlib/mscorlib.shared.sources.props

```diff
@@ -804,6 +804,7 @@
     <IoSources Include="$(BclSourcesRoot)\System\IO\ReadLinesIterator.cs" />
     <IoSources Include="$(BclSourcesRoot)\System\IO\SeekOrigin.cs" />
     <IoSources Include="$(BclSourcesRoot)\System\IO\Stream.cs" />
+    <IoSources Include="$(BclSourcesRoot)\System\IO\StreamHelpers.cs" />
```

Comment:
It should use the exact same name as in corefx (`StreamHelpers.CopyValidation.cs`).